pages/weatherDetail.tsx

```diff
@@ -62,7 +62,7 @@ function WeatherDetail() {
           marginRight="auto"
           marginLeft="auto"
         >
-          <Link href="/weather">
+          <Link href="/Weather">
```

Comment:
Typically we keep paths all lowercase. This is a holdover from ASP.Net days and its not "wrong", but you won't see it at Bushel.